src/components/Contacts/ContactDetails/ContactDetailsTab/Tags/ContactTags.tsx

```diff
@@ -95,24 +99,25 @@ export const ContactTags: React.FC<ContactTagsProps> = ({
   };
 
   const onSubmit = async (
-    { tag }: { tag: string },
-    { resetForm }: FormikHelpers<{ tag: string }>,
+    { tagList }: { tagList: string[] & never[] },
```

Comment:
What's the `& never[]` about?

Reply:
So every time the user adds new tags, I want to reset the initial Formik value of ```tagList``` back to an empty array:
```
     <Formik
        initialValues={{ tagList: [] }}
        validationSchema={tagSchema}
        onSubmit={onSubmit}
      >
```
So it's inheriting the type of ```never[]``` from that initial value, plus the ```tagList: string[]``` I set.

Reply:
Were the types complaining? `string[]` can also be an empty array. But this if fine if it's needed to work :) I'm not familiar with Formik's typings.

Reply:
It complained in the onSubmit function. That's what I thought too, but TS just kept yelling 🤷‍♂️